docs/getting_started/basics/contexts.md

```diff
@@ -0,0 +1,15 @@
+# Context lookup
```

Comment:
New files also have to be listed here: https://github.com/LinkedSoftwareDependencies/Documentation-Components.js/blob/master/mkdocs.yml#L5